### PR TITLE
linking bitcoin wiki / units

### PR DESCRIPTION
**Pull request by s72785**
and listing finney

### CODE REVIEW

File: _data/devdocs/en/guides/payment_processing.md [hunk 1]
```diff
@@ -153,16 +153,18 @@ You must pay by: 2014-04-01 at 23:00 UTC
 
 Indicating the denomination is critical. As of this writing, popular
 Bitcoin wallet software defaults to denominating amounts in either bitcoins (BTC)
-, millibitcoins (mBTC) or microbitcoins (uBTC, "bits"). Choosing between each unit is widely supported,
+, millibitcoins (mBTC) or microbitcoins (uBTC, "bits").
+Choosing between each unit is widely supported,
 but other software also lets its users select denomination amounts from
-some or all of the following options:
+some preselected or all [possible options](https://en.bitcoin.it/wiki/Units):
```
Comment (luke-jr) [inline _data/devdocs/en/guides/payment_processing.md]: The wording here seems to imply the table below includes all options, which is far from complete.

Comment (luke-jr) [reply _data/devdocs/en/guides/payment_processing.md]: (This PR is probably an *improvement* over the status quo, however)

Comment (s72785) [reply _data/devdocs/en/guides/payment_processing.md]: also thought of adding all decimals to the table, but wasnt sure thats helpful, is it? Or what wording would be more appropriate to not incline the table below includes all possibilities?

**Comment by wbnns** [comment]
@s72785 Thanks!

**Pull request merged**